hbase-shell/src/main/ruby/hbase/admin.rb

```diff
@@ -1043,6 +1043,7 @@ def status(format, type)
       else
         puts "1 active master, #{cluster_metrics.getBackupMasterNames.size} backup masters,
               #{cluster_metrics.getLiveServerMetrics.size} servers,
+              #{@admin.listDecommissionedRegionServers.size} decommissioned,
```

Comment:
We do not have this information in cluster metrics?

Reply:
Thanks @Apache9 for your review.

 Now, we do not have `decommissionedRegionServers` in cluster metrics. But we have metrics `numDrainingRegionServers`  and `drainingRegionServers`. 

There is a method `listDecommissionedRegionServers` in `Admin`,  so called directly here.

Reply:
Then let's add it to the cluster metrics?

Reply:
Thanks for your comment. 

At first, I wanted to add `drainingRegionServers` to cluster metrics, but I found that `drainingRegionServers` already exist. In most cases, `drainingRegionServers` and `decommissionedRegionServers` may represent the same list of regionservers, so I didn't add this metric.

Reply:
Cluster metrics approach is better than calling additional admin API (includes a separate RPC). 

Decommissioned and draining RegionServers are same,
https://github.com/apache/hbase/blob/1be2c83b3dc8306050f67828c9c2e361f19481c6/hbase-server/src/main/java/org/apache/hadoop/hbase/master/HMaster.java#L3821

Add decommissioned/draining RegionServer to ClusterMetrics class, currently it is not there.

Reply:
> Cluster metrics approach is better than calling additional admin API (includes a separate RPC).
> 
> Decommissioned and draining RegionServers are same,
> 
> https://github.com/apache/hbase/blob/1be2c83b3dc8306050f67828c9c2e361f19481c6/hbase-server/src/main/java/org/apache/hadoop/hbase/master/HMaster.java#L3821
> 
> Add decommissioned/draining RegionServer to ClusterMetrics class, currently it is not there.

Thanks @pankaj72981 for your comments. I found `drainingRegionServers` already exist, we can see HBASE-25460. Do you mean to change `drainingRegionServers` to `decommissionedRegionServers `?

Reply:
HBASE-25460 added draininigRegionServers as a JMX metric. Can you access it via ClusterMetrics class?

Reply:
> [HBASE-25460](https://issues.apache.org/jira/browse/HBASE-25460) added draininigRegionServers as a JMX metric. Can you access it via ClusterMetrics class?

Thanks @pankaj72981 for your comment. 

I will try to add this to ClusterMetrics. Do you think `draininigRegionServers` or `decommissionedRegionServers` more appropriate? @Apache9 Please also take a look. Thank you.